scaleway/table_scaleway_registry_image.go

```diff
@@ -0,0 +1,245 @@
+package scaleway
+
+import (
+	"context"
+
+	"github.com/scaleway/scaleway-sdk-go/api/registry/v1"
+
+	"github.com/scaleway/scaleway-sdk-go/scw"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableScalewayRegistryImage(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:              "scaleway_registry_image",
+		Description:       "Registry Images allow you to manage your docker images in Scaleway.",
+		GetMatrixItemFunc: BuildRegionList,
+		List: &plugin.ListConfig{
+			Hydrate:       listRegistryImages,
+			ParentHydrate: listRegistryNamespaces,
+			KeyColumns: []*plugin.KeyColumn{
+				{
+					Name:    "name",
+					Require: plugin.Optional,
+				}, {
+					Name:    "namespace_id",
+					Require: plugin.Optional,
+				},
+			},
+		},
+		Get: &plugin.GetConfig{
+			Hydrate:    getRegistryImage,
+			KeyColumns: plugin.AllColumns([]string{"id"}),
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The Image name, unique in a namespace",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "namespace_id",
+				Description: "The unique ID of the Namespace the image belongs to",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("NamespaceID"),
+			},
+			{
+				Name:        "id",
+				Description: "A unique identifier of the instance.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ID"),
+			},
+			{
+				Name:        "status",
+				Description: "The current status of the Image.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Status").Transform(transform.ToString),
+			},
+			{
+				Name:        "visibility",
+				Description: "A `public` image is pullable from internet without authentication, opposed to a `private` image. `inherit` will use the namespace `is_public` parameter.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Visibility").Transform(transform.ToString),
+			},
+			{
+				Name:        "size",
+				Description: "Image size in bytes, calculated from the size of image layers. One layer used in two tags of the same image is counted once but one layer used in two images is counted twice.",
+				Type:        proto.ColumnType_INT,
+				Transform:   transform.FromField("Size").Transform(transform.ToInt),
```

Comment:
Do we need this transform function? By default, we transform the columns to camel case which should transform `Size` to `size`. Also, I see the column's datatype is already an integer, any specific reason why we need to transform it to `INT` again?